.abapgit.xml

```diff
@@ -10,6 +10,7 @@
     <item>/CONTRIBUTING.md</item>
     <item>/LICENSE</item>
     <item>/README.md</item>
+    <item>*.orig</item>
```

Comment:
add "/" to only ignore these files if they are in the root? in case there is a MIME object with .orig

I am not familiar with .orig files, what are they used for?